pisa/stages/osc/nsi_params.py

```diff
@@ -115,3 +137,560 @@ def eps_tautau(self, value):
         assert isinstance(value, self.eps_type)
         self._eps_matrix[2][2] = self._eps_tautau = value
 
+
+class StdNSIParams(NSIParams):
+    """
+    NSI parameters in the standard parameterization.
+
+    Attributes
+    ----------
+    eps_matrix : 2d float array of shape (3, 3)
+        Effective NSI coupling matrix.
+
+    eps_ee, eps_emu, eps_etau, eps_mumu, eps_mutau, eps_tautau : float or complex
+        Effective NSI coupling parameters.
+
+    """
+
+    def __init__(self):
+        super(StdNSIParams, self).__init__(0.,(0.,0.),(0.,0.),0.,(0.,0.),0.)
+        '''self._eps_ee = 0.
+        self._eps_emu = 0.
+        self._eps_etau = 0.
+        self._eps_mumu = 0.
+        self._eps_mutau = 0.
+        self._eps_tautau = 0.'''
+
+    # --- NSI epsilons ---
+    @property
+    def eps_ee(self):
+        """effective nue-nue NSI coupling parameter"""
+        return self.eps_matrix[0, 0].real
+
+    @eps_ee.setter
+    def eps_ee(self, value):
+        part1 = self.eps_matrix[0, 0].imag
+        part2 = 1.j * part1
+        part3 = value + part2
+        self.eps_matrix[0, 0] = part3
```

Comment:
Is this duplication of setting the matrix element intended?

Reply:
no, I'll remove the less verbose version